src/internal/nft-api.ts

```diff
@@ -378,6 +379,19 @@ export async function computeRarity(
   return getNftRarityFromRaw(response);
 }
 
+export async function summarizeNftAttributes(
+  config: AlchemyConfig,
+  contractAddress: string,
+  srcMethod = 'summarizeNFTAttributes'
```

Comment:
can you rename to `summarizeNftAttributes`? the goal is to have the original name of the sdk method.

Reply:
Yup!